include/wx/overlay.h

```diff
@@ -17,6 +17,8 @@
     #define wxHAS_NATIVE_OVERLAY 1
 #elif defined(__WXOSX__) && wxOSX_USE_COCOA
     #define wxHAS_NATIVE_OVERLAY 1
+#elif defined(__WXGTK3__) // We should test for GTK_CHECK_VERSION(3,18,0)
```

Comment:
This needs to be done at configure level, similarly to what we do for e.g. `__WXGTK218__`, i.e. we need to define `__WXGTK318__` there which would be available in this header (unlike GTK version-checking macros).

Reply:
Can you show me how to do that, please ?

Reply:
Added in this commit e665298.